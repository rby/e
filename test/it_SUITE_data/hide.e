{
  a := 1
  b:= "some string"

}
