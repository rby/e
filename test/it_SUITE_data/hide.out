[{hide,[{':=',{identifier,{2,2},"a"},{integer,{2,7},1}},
                {':=',{identifier,{3,2},"b"},
                      {identifier,{3,6},"some string"}}]}]
